tests/cypress/tests-e2e/legacyParamImport.cy.js

```diff
@@ -411,4 +440,142 @@ describe('Test on legacy param import', () => {
             cy.readStoreValue('state.position.projection.epsgNumber').should('eq', 3857)
         })
     })
+
+    context('Extra Parameter Imports', () => {
+        it('sets the language correctly', () => {
+            cy.goToMapView({ lang: 'it' }, false)
+            cy.readStoreValue('state.i18n.lang').should('eq', 'it')
+        })
+
+        it('shows the correct crosshair', () => {
+            function checkCrosshair(crossHair) {
+                cy.readStoreValue('state.position').then((positionStore) => {
```

Comment:
Generally we should avoid such construct in cypress `cy.readStoreValue('state.position').then()` this has for sure no retry and can lead to flaky test unless we are sure that the value is set before the call is done (due to other waits). Basically the following construct should be prefered
`cy.readStoreValue('state.position').should()` which will retry until the assertion in should is done (note we can do assertion in a callback in should as well)